store/modules/events.ts

```diff
@@ -52,8 +66,12 @@ export default {
     }
   },
   actions: {
-    async fetchEvents () {
-      const eventsModule = await import('~/content/events/events-previews.json')
+    async fetchFutureEvents () {
```

Comment:
Curious, I would have expected to fetch just one list, then filter by date. However, since we are caching the lists once a day, I don't see anything bad with this approach. :+1: